app/src/main/java/live/hms/app2/ui/meeting/MeetingViewModel.kt

```diff
@@ -1370,8 +1370,12 @@ class MeetingViewModel(
         hmsSDK.startHLSStreaming(config, object : HMSActionResultListener {
             override fun onError(error: HMSException) {
                 viewModelScope.launch {
-                    _events.emit(Event.Hls.HlsError(error))
-                    hlsToggleUpdateData.postValue(false)
+                    if (error.message.contains("invalid input")) {
+                        startHls(hlsUrl, recordingConfig, true)
```

Comment:
What happens If `invalid input` error comes even after starting HLS without `null` meetingVariants? Then this is infinite loop?

Reply:
actually when i checked with ravi .. its an error that only comes when destinations are null